core/DataTable/Filter/ExcludeLowPopulation.php

```diff
@@ -68,7 +68,7 @@ public function __construct($table, $columnToFilter, $minimumValue, $minimumPerc
 
         $this->columnToFilter = $this->selectColumnToExclude($columnToFilter, $row);
 
-        if ($minimumValue == 0) {
+        if ($minimumValue === 0) {
```

Comment:
based on the doc block `$minimumValue` can be `false`. This case was covered here as well. So guess this would be better:
```suggestion
        if ((int) $minimumValue === 0) {
```